test/Host/TestUtils/RepoDirectoriesProvider.cs

```diff
@@ -31,7 +31,7 @@ public class RepoDirectoriesProvider
             string corehostPackages = null,
             string corehostDummyPackages = null)
         {
-            _repoRoot = repoRoot ?? Path.Combine(Directory.GetCurrentDirectory(), "..", "..");
+            _repoRoot = repoRoot ?? Path.Combine(Directory.GetCurrentDirectory(), "..", "..", "..");
```

Comment:
In hindsight, CWD isn't a fantastic thing to depend on, any ideas on how to improve this?